includes/class-amp-post-type-support.php

```diff
@@ -123,7 +123,12 @@ public static function get_support_errors( $post ) {
 			$post = get_post( $post );
 		}
 		$errors = [];
-
+		
+		if ( ! isset( $post->post_type ) && ! isset( $post->ID ) ) {
+			$errors[] = 'post-type-support';
```

Comment:
I think a different error ID should be used here. Maybe something like `invalid-post`. Thoughts @westonruter?

Reply:
Yes, `invalid-post` would be more appropriate. We should then make sure that this new value is accounted for in places where this code is invoked.

Reply:
Awesome, thank you guys.. Adding this code to my sites.

Reply:
>  We should then make sure that this new value is accounted for in places where this code is invoked.

Done so in https://github.com/ampproject/amp-wp/pull/5871/commits/6b9a10d2e97415ac3a7f11ae13b7ba706264ebbb.